addons/common/fnc_updatePerFrameHandlerDelay.sqf

```diff
@@ -0,0 +1,34 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_updatePerFrameHandlerDelay
+
+Description:
+    Updates the delay of an existing perFrameHandler.
+
+Parameters:
+    _handle   - The existing perFrameHandler's handle. <NUMBER>
+    _delay    - The amount of time in seconds between executions, 0 for every frame. (optional, default: 0) <NUMBER>
+
+Returns:
+    true if successful, false otherwise <BOOLEAN>
+
+Examples:
+    (begin example)
+        _wasSuccessful = [_handle, _newDelay] call CBA_fnc_updatePerFrameHandlerDelay;
+    (end)
+
+Author:
+    Mokka, OverlordZorn
+---------------------------------------------------------------------------- */
+
+params [["_handle", -1, [0]], ["_newDelay", 0, [0]], ["_updateExectionTime", false, [false]]];
+
+private _idx = GVAR(PFHhandles) param [_handle];
```

Comment:
need to be wrapped in `CBA_fnc_directCall` or idx could change

Reply:
the whole thing or just those lines?